code/modules/nano/interaction/default.dm

```diff
@@ -43,6 +43,8 @@
 		var/obj/machinery/Machine = src_object
 		if(Machine.emagged)
 			return STATUS_INTERACTIVE
+	if(istype(src_object, /obj/item/device/radio/borg/syndicate)) //Not ideal, but they can't access their own radio otherwise.
```

Comment:
On Bay Syndicate borgs no longer has special NanoUI treatment. Fairly big changes for it to work tho, see https://github.com/Baystation12/Baystation12/pull/10694.